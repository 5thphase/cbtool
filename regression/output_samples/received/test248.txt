Background operation execution success. Operation "vm-detach" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.1.250,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.1.250,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,vm_5,false --operation=vm-detach --cn=TESTCLOUD --uuid=033D3751-876F-54FF-9FE4-07EA02E951FF --daemon". The process id is 23180.

Going to unconditionally wait for 10s (10 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (10.0%)
Waited 2 seconds... (20.0%)
Waited 3 seconds... (30.0%)
Waited 4 seconds... (40.0%)
Waited 5 seconds... (50.0%)
Waited 6 seconds... (60.0%)
Waited 7 seconds... (70.0%)
Waited 8 seconds... (80.0%)
Waited 9 seconds... (90.0%)
Waited 10 seconds... (100.0%)
Waited for 10 seconds.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |156.4.116.25    |simhostd1                  |SUT                 |none    |none       |776531B5-7DCD-5B1C-BDD2-0962E8F87DF2
|vm_3          |tinyvm              |pico32      |108.252.120.109 |simhostb1                  |SUT                 |none    |none       |1304A629-B272-53F3-99F2-C116650A5340
|vm_6          |tinyvm              |pico32      |110.213.95.178  |simhostd1                  |SUT                 |none    |none       |74AF7F16-9A25-5973-9371-431B14E3499D
|vm_7          |tinyvm              |pico32      |2.81.152.255    |simhostd0                  |SUT                 |none    |none       |BC432A15-7D32-51E6-966C-387BE1E0F1EE
|vm_8          |tinyvm              |pico32      |252.35.149.231  |simhostd2                  |SUT                 |none    |none       |4A86138A-F12F-50FF-9C62-1042B8432A16
|vm_9          |db2                 |platinum64  |146.116.158.219 |simhostb0                  |SUT                 |none    |none       |C6E7C4CD-8A47-5826-A061-8C1670692FE2
|vm_10         |netclient           |iron32      |229.61.116.135  |simhosta2                  |SUT                 |none    |none       |66A867D7-232A-59C5-A60D-E6D998D88BA1
|vm_11         |fen_hpc             |platinum64  |139.97.26.132   |simhosta4                  |LG                  |none    |none       |7B4DB42B-F55F-5528-8E29-181D72F2098C
|vm_12         |tinyvm              |iron32      |187.148.255.210 |simhostd2                  |SUT                 |none    |none       |447FB2A8-F053-5C88-AE6D-FD9F19319362
|vm_13         |tinyvm              |iron32      |147.178.87.142  |simhosta1                  |SUT                 |none    |none       |D034CB71-8EDC-52AA-BEA1-D4783A802FD6
|vm_14         |predictablevm       |iron32      |1.2.3.4         |simhosta0                  |SUT                 |none    |none       |A23F8E82-5D27-5624-AD48-360368A4B6F3
|vm_15         |tinyvm              |pico32      |144.182.108.210 |simhostd2                  |SUT                 |none    |none       |A2FAF47B-6AAF-50A3-AE4B-5805BB7D64CD
|vm_16         |tinyvm              |pico32      |68.190.243.183  |simhostd2                  |SUT                 |none    |none       |D67B6962-A80A-5060-8189-5FA56762E6C9
|vm_17         |tinyvm              |pico32      |73.86.251.20    |simhostd0                  |SUT                 |none    |none       |25DCEEC3-AAFF-5AD0-9DBD-EA815E88D8F8
|vm_18         |tinyvm              |pico32      |162.191.47.183  |simhostd3                  |SUT                 |none    |none       |7F75D423-84E8-50C5-8B1C-741A46E3D06A
|vm_19         |tinyvm              |pico32      |206.160.88.160  |simhosta0                  |SUT                 |none    |none       |82FEB23E-38BC-5F8D-841F-5A68BF2A6FB1
|vm_20         |tinyvm              |pico32      |17.187.196.11   |simhosta0                  |SUT                 |none    |none       |D8825ACB-3DA1-593E-928F-91C48A6E8A68
|vm_21         |tinyvm              |pico32      |104.137.146.56  |simhostc3                  |SUT                 |none    |none       |48A09473-7E1A-5800-8530-891ADB708644
|vm_22         |tinyvm              |pico32      |218.86.37.104   |simhostb2                  |SUT                 |none    |none       |C4C76CC2-7DE2-5DBA-B0A2-A1986EFFF633

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.98M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |3792449                       
|Total Connections Received                           |81187                         
|Total Commands Processed                             |5784275                       
|Number of Keys                                       |430                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |1339392                       
|Data Size                                            |168624                        
|Index Size                                           |294336                        
|Average Object Size                                  |401.485714286                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |19                            
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |19                            
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |3                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |113                           

 ---------------- VMC-WIDE COUNTERS ----------------
|76A8BFA3-0C2E-5D7E-A667-254C804D93C2 (simzone_b) VM RESERVATIONS|5                             
|785C590F-E88D-5ABB-9A6F-08292348ADB9 (simzone_c) VM RESERVATIONS|5                             
|DE75759E-CFAB-53C8-A062-05C5AB16764D (simzone_a) VM RESERVATIONS|8                             
|706D24CD-9D55-592D-915D-7D4D41F35312 (simzone_d) VM RESERVATIONS|1                             

