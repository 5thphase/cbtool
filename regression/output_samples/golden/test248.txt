Background operation execution success. Operation "vm-detach" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.1.250,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.1.250,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,vm_5,false --operation=vm-detach --cn=TESTCLOUD --uuid=9A9038FF-525E-5953-A91F-BA8B51E0629D --daemon". The process id is 17982.

Going to unconditionally wait for 10s (10 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (10.0%)
Waited 2 seconds... (20.0%)
Waited 3 seconds... (30.0%)
Waited 4 seconds... (40.0%)
Waited 5 seconds... (50.0%)
Waited 6 seconds... (60.0%)
Waited 7 seconds... (70.0%)
Waited 8 seconds... (80.0%)
Waited 9 seconds... (90.0%)
Waited 10 seconds... (100.0%)
Waited for 10 seconds.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |29.82.87.242    |simhosta1                  |SUT                 |none    |none       |6ACE62BD-D7F9-5736-8A15-14EFCDB4D3C8
|vm_3          |tinyvm              |pico32      |70.169.109.31   |simhostb1                  |SUT                 |none    |none       |DD958086-32D4-5B48-9558-8A8EC2E9A164
|vm_6          |tinyvm              |pico32      |7.189.195.208   |simhosta1                  |SUT                 |none    |none       |4467B178-CAEC-521A-96DD-477E02D7A7DB
|vm_7          |tinyvm              |pico32      |98.157.96.103   |simhostb1                  |SUT                 |none    |none       |E83EE478-7F8E-5E8D-B913-82EAF311CE79
|vm_8          |tinyvm              |pico32      |3.120.74.54     |simhosta1                  |SUT                 |none    |none       |B57980AF-27E4-5B16-95D2-A3C9C8709AC6
|vm_9          |db2                 |platinum64  |78.217.132.82   |simhosta1                  |SUT                 |none    |none       |21F523CA-5683-5874-8DFF-FE0B3E68B77E
|vm_10         |netclient           |iron32      |14.179.174.236  |simhosta2                  |SUT                 |none    |none       |2AB6AB34-A741-50D2-9F09-BD31BA45C80C
|vm_11         |fen_hpc             |platinum64  |56.93.146.218   |simhostd1                  |LG                  |none    |none       |4392830F-431A-5757-BEF2-789A49CD997D
|vm_12         |tinyvm              |iron32      |183.128.82.165  |simhostb1                  |SUT                 |none    |none       |B1D912F0-2781-50DE-A02A-AB857CB9A235
|vm_13         |tinyvm              |iron32      |28.51.38.153    |simhostb1                  |SUT                 |none    |none       |9E1BE97A-DD71-5BDB-9930-9903AA014075
|vm_14         |predictablevm       |iron32      |1.2.3.4         |simhostb1                  |SUT                 |none    |none       |A6BE1AB6-F90E-5F1D-8BAF-C029580346F0
|vm_15         |tinyvm              |pico32      |203.131.96.243  |simhostc1                  |SUT                 |none    |none       |520DCF94-E762-5CFD-8EC5-33ADC8097D49
|vm_16         |tinyvm              |pico32      |221.230.219.235 |simhostc1                  |SUT                 |none    |none       |38E550F9-B6EE-5E04-B3B0-505C8B4046E1
|vm_17         |tinyvm              |pico32      |190.253.151.19  |simhostc1                  |SUT                 |none    |none       |648637A9-2DED-5BE7-8642-C6CE843565B3
|vm_18         |tinyvm              |pico32      |39.43.67.135    |simhosta1                  |SUT                 |none    |none       |15052811-FFAE-5704-AB00-FFDDA1AB76AC
|vm_19         |tinyvm              |pico32      |238.205.206.139 |simhostc1                  |SUT                 |none    |none       |A477B2AF-E8E2-5036-8515-F48CF4B0851C
|vm_20         |tinyvm              |pico32      |43.199.181.43   |simhostb1                  |SUT                 |none    |none       |0587EEDA-5A3B-548C-BE72-4095113C207B
|vm_21         |tinyvm              |pico32      |243.45.44.65    |simhostc1                  |SUT                 |none    |none       |809642D0-08AF-5793-92BC-C4FD5023A60A
|vm_22         |tinyvm              |pico32      |73.83.105.151   |simhostb1                  |SUT                 |none    |none       |F4E55870-AD0C-50E9-88D3-4EA1D6144A89

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.85M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |507690                        
|Total Connections Received                           |10481                         
|Total Commands Processed                             |594666                        
|Number of Keys                                       |423                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |1282048                       
|Data Size                                            |168640                        
|Index Size                                           |294336                        
|Average Object Size                                  |401.523809524                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |19                            
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |19                            
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |3                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |113                           

 ---------------- VMC-WIDE COUNTERS ----------------
|49282BA1-6D05-5C43-9AF0-FFE568AFD5F9 (simzone_b) VM RESERVATIONS|7                             
|F04BDEFB-2E30-560C-802C-A1FABC5D8D0C (simzone_a) VM RESERVATIONS|6                             
|9C8B8765-2143-577F-9E2E-DEDF96009D33 (simzone_d) VM RESERVATIONS|1                             
|68F26DD5-96FF-5310-9ED9-3568A17CBF4A (simzone_c) VM RESERVATIONS|5                             

